### ISSUE TITLE
Active alternatives to react-planner?

### ISSUE THREAD

**Issue by LeoPapais** [cvdlab/react-planner#240]
Hey guys,

That's a nice project, but it seems to be inactive for a while now. Anyone can point me to more active alternatives in the same space (2D or 3D floorplan drawing tools)?

I've found BlueprintJS (original https://github.com/furnishup/blueprint3d and fork https://github.com/aalavandhaann/blueprint-js ) but it doesn't seem to be active too.

Any help is appreciated,
Thanks in advance

**Comment by nikakhachi** [comment]
@LeoPapais Did you find something ?

**Comment by crfloyd** [comment]
I too am looking for something more active like this. It would be nice to hear something from one of the contributors regarding the current state of this project's activity. Im searching for a candidate library to include in a planning project and I'd be happy to make significant contributions to this if others have interest in keeping it active.

**Comment by LeoPapais** [comment]
> @LeoPapais Did you find something ?

No, I never found anything. So I started implementing something custom, and, unfortunately, I can't share it :/

**Comment by yangxyo** [comment]
Did you find something ?

**Comment by GeoloeG-IsT** [comment]
We are actively working on a migration from this repo for our own need (www.luccid.ai) and will be releasing this fork as open source soon. If you wanna help us, please contact me!

**Comment by test-user1-IIO** [comment]
@MeTaNoV any update on the progress? I too am looking into porting the fork from https://github.com/aalavandhaann/blueprint-js  to current standards.

**Comment by matteocollina** [comment]
@test-user1-IIO did you ported the project into something newest?
